acceptance/suites/pre_suite/compat_common/30_install_dev_repos.rb

```diff
@@ -1,4 +1,4 @@
-install_opts = options.merge({ :dev_builds_repos => ["PC1"]})
+install_opts = options.merge({ :dev_builds_repos => ["puppet"]})
```

Comment:
I'm digging into beaker re: this variable, and it looks like this isn't even used.